### PR TITLE
cmake: add 3.25.3 + 3.24.4

### PR DESCRIPTION
**Pull request by Croydon**
- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by Croydon** [comment]
> 🤖 **Beep Boop**! This pull request is making changes to 'recipes/cmake/_/_'.
> 
> 👋 @Croydon you might be interested. 😉

I'm very interested. Thank you!

File: recipes/cmake/3.x.x/conandata.yml [hunk 1]
```diff
@@ -14,9 +14,9 @@ sources:
   "3.23.5":
     url: "https://github.com/Kitware/CMake/releases/download/v3.23.5/cmake-3.23.5.tar.gz"
     sha256: "f2944cde7a140b992ba5ccea2009a987a92413762250de22ebbace2319a0f47d"
-  "3.24.3":
-    url: "https://github.com/Kitware/CMake/releases/download/v3.24.3/cmake-3.24.3.tar.gz"
-    sha256: "b53aa10fa82bff84ccdb59065927b72d3bee49f4d86261249fc0984b3b367291"
-  "3.25.2":
-    url: "https://github.com/Kitware/CMake/releases/download/v3.25.2/cmake-3.25.2.tar.gz"
-    sha256: "c026f22cb931dd532f648f087d587f07a1843c6e66a3dfca4fb0ea21944ed33c"
```
Comment (SpaceIm) [inline recipes/cmake/3.x.x/conandata.yml:22]: don't remove yet please, it's a mess to manage in conancenter in recipes depending on CMake.

**Review by prince-chrismc — APPROVED** [review]

**Review by ericLemanissier — APPROVED** [review]

**Comment by SpaceIm** [comment]
Should it be closed due to https://github.com/conan-io/conan-center-index/pull/16503 ?

**Comment by prince-chrismc** [comment]
Yes, plus it's merge conflicts, I will tick this as closed

**Pull request closed without merging**